schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
 	date_of_birth DATE NOT NULL,
 	escape_attempt INT,
 	neutered BOOLEAN NOT NULL,
-	weight_kg DECIMAL NOT NULL
+	weight_kg DECIMAL(10, 2) NOT NULL,
+	species varchar(100)
```

Comment:
**Changes Required**
- [ ] This will work if we were creating the tables for the first time, otherwise we will need to `DROP` the table first if it `exists` before adding it directly. A better way is to use `ALTER TABLE animals ADD species varchar(255);`

Reply:
Hey Emanuel Okello,
What I did not add the species directly, I DROP the table then Create it again and it's the same as if I add a new collon by using `ALTER TABLE animals ADD species varchar(255);` which is the best and quick what to do that
`DROP TABLE animals;`

`CREATE TABLE animals (
id BIGSERIAL NOT NULL PRIMARY KEY,
name VARCHAR(100) NOT NULL,
date_of_birth DATE NOT NULL,
escape_attempts INT,
neutered BOOLEAN NOT NULL,
weight_kg DECIMAL(10, 2) NOT NULL,
species VARCHAR(100)
);
`